src/structures/User.js

```diff
@@ -179,17 +179,19 @@ class User extends Base {
    * @readonly
    */
   get hexAccentColor() {
-    if (!this.accentColor) return null;
+    if (!this.accentColor && typeof this.accentColor !== 'number') return this.accentColor;
```

Comment:
Couldn't you simplify this to just the typeof check?

Reply:
I’d rather not because people could set the accentColor on their own and thus break it a bit, I think this is more secure

Reply:
I don't think we do this anywhere else, do we?
Generally, if you mess with properties you aren't supposed to mess with, expect weird behavior. 🤷

Reply:
Sure then I guess